backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/UserService.cs

```diff
@@ -64,12 +64,23 @@ private async Task<User> GetUserById(long id)
 
         public async Task<UserDto> CreateUserPreferences(NewUserDto userDto)
         {
+            if (userDto is null)
```

Comment:
If parameter is not nullable, then exception will be probably thrown automatically?